app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :post, class_name: 'User'
+
```

Comment:
- [ ] I believe here you actually wanted to say `belongs_to :author, class_name: 'User`. Kindly fix the error 🙏